x-pack/plugins/case/common/api/cases/status.ts

```diff
@@ -21,6 +23,16 @@ export const CaseStatusRt = rt.union([
 
 export const caseStatuses = Object.values(CaseStatuses);
 
+export const CaseStatusFilterRt = rt.union([
+  rt.literal(AllCaseType),
+  rt.literal(CaseStatuses.open),
```

Comment:
I think we can do it like `export const CaseStatusFilterRt = rt.union([rt.literal(AllCaseType), ...CaseStatusRt.types]);` and avoid the duplication.